portal/static/js/admin.js

```diff
@@ -198,9 +181,9 @@ AdminTool.prototype.abortRequests = function(callback) {
 AdminTool.prototype.getUserIdArray = function(_userIds) {
   var us = "", ct = 0, arrUsers = [];
   if (!_userIds) {
-     _userIds = this.getInitUserList();
+     _userIds = this.getPatientsIdList();
   };
-  var max_ct = Math.max(__patients_list.length/10, 25);
+  var max_ct = Math.max(_userIds.length/10, 10);
```

Comment:
Is this what you want?  Won't a really large `_userIds` lead to a value greater than 10?